modules/precisoBidAdapter.md

```diff
@@ -0,0 +1,67 @@
+# Overview
+
+Module Name: Preciso Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: tech@preciso.net
+
+
+# Description
+
+Module that connects to Preciso's demand sources.
+
+# Test Parameters
+```
+    var adUnits = [
+           // banner
+           {
+               code: 'test-div',
+               mediaTypes: {
+	           banner: {
+                       sizes: [[300, 250]],
+                   }
+               },
+               bids: [
+                   {
+                       bidder: "preciso",
+                       params: {
+                           region: 'prebid-eu',
+                           publisherId: 'PREBID_TEST_ID',
```

Comment:
Please update the publisherId to a working id for testing, thank you!

Reply:
We referred to the RTB House bid adapter while creating ours. The RTB House bidder used The same publisher ID. Is the publisher ID incorrect for testing?

Reply:
Ideally we'd like to see a publisher Id that your endpoint will return a bid with so that we can confirm your adapter packages requests and responds to your bids properly.

Reply:
This still needs to be updated, thank you